src/WallGo/Config/WallGoDefaults.ini

```diff
@@ -79,3 +83,10 @@ absoluteTol = 1e-10
 # minimum and maximum temperature obtained in the template model.
 tmax = 1.2
 tmin = 0.8
+
+[BoltzmannSolver]
```

Comment:
I agree that this is useful (as the current situation shows), but can we warn the user more explicitly? E.g.: before the rest of the description:

WARNING: THIS CHANGES THE COLLISION TERMS WRT TO THEIR PHYSICAL VALUE